stylesheets/app.scss

```diff
@@ -2085,3 +2085,12 @@ h2.home-title {
     }
   }
 }
+
+// Hacking for drupal
```

Comment:
@LightGuard Could you please document in the comment here what this works around in Drupal so that if we ever want to remove it in the future, we can work out what it is doing.